README.md

```diff
@@ -33,3 +33,67 @@ The `git clonefork` can be used by running `git clonefork {REMOTE_ORIGIN_URL}` w
 ## Installation
 To install the utils, use the `./install.sh` script provided in this repo. If you would like to specify which util to install rather than install all of them, you can
 use `./install.sh -n <name_of_util`. For example, to install `git-sync` only, you could run `./install.sh -n git-sync`.
+
+
+## How to set up an SSH Key with GitHub
```

Comment:
I'm thinking we can add setting up SSH key with GitHub/GitLab into a "troubleshooting" section in the README, rather than making it it's own section.